test/index.test.js

```diff
@@ -896,6 +888,7 @@ describe('Segment.io', function() {
 
             analytics.assert(res.crossDomainId === 'xdomain-id-1');
             analytics.assert(res.fromDomain === 'xid.domain2.com');
+            analytics.assert.equal(res.timestamp, 1540432000000);
```

Comment:
nit-pick: let's either use `analytics.assert(foo === bar)` or `analytics.assert.equal(foo, bar)` in this scope for consistency

Reply:
I had this originally to match the style, but then my test was failing with a very unhelpful error message (true !== false). I think it'll help to be better to be inconsistent and provide a more helpful error message.

Do you feel strongly one way or the other?

Reply:
Oh, makes sense. Let's keep it then